pkg/virt-handler/options_test.go

```diff
@@ -0,0 +1,282 @@
+/*
+Copyright 2024 The KubeVirt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package virthandler
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	cmdv1 "kubevirt.io/kubevirt/pkg/handler-launcher-com/cmd/v1"
+	"kubevirt.io/kubevirt/pkg/virt-handler/node-labeller/api"
+)
+
+var _ = Describe("Parsing VMI Options", func() {
+	const (
+		memorySize = 2 * 1024 * 1024
+		memoryUnit = "KiB"
+	)
+
+	Context("virt-handler VM processes VMI options during update", func() {
+		It("should convert api capabiliites to cmdv1 topology with single cell", func() {
+			caps := &api.Capabilities{}
+			caps.Host.Topology.Cells = api.Cells{
+				Cell: []api.Cell{
+					{
+						Memory: api.Memory{Unit: memoryUnit, Amount: memorySize},
```

Comment:
nit: Shouldn't amount here be equal to `16256896` if we try to follow `pkg/virt-handler/node-labeller/testdata/capabilities.xml`?

Reply:
The reason behind my request is so future reviewers will not have to guess where do these numbers came from.
I think it would also be a good idea to state that the tests follow the existing test data in the commit message.

Reply:
Done.